acis_thermal_check/test_utils.py

```diff
@@ -1,4 +1,3 @@
-from ..utils import TASK_DATA
```

Comment:
Probably doesn't matter that much, but with the new name `test_utils.py`, pytest will see this as a potential tests file and try to discover / run tests in it.  I think that nothing inside matches the test discovery rules.